### ISSUE TITLE
Expiry of data on s3 using vmbackup

### ISSUE THREAD

**Issue by salarali** [VictoriaMetrics/VictoriaMetrics#4022]
### Is your question request related to a specific component?

vmbackup 

### Describe the question in detail

I am utilizing vmbackup to incrementally backup the database to s3. I have a retention of 90 days set on vmstorage. If I set the expiry of data on s3 to be 90 days as well, will that work when I try to run vmrestore? 

### Troubleshooting docs

- [ ] General - https://docs.victoriametrics.com/Troubleshooting.html
- [ ] vmagent - https://docs.victoriametrics.com/vmagent.html#troubleshooting
- [ ] vmalert - https://docs.victoriametrics.com/vmalert.html#troubleshooting

**Comment by zekker6** [comment]
Yes, that's right. If you could share vmbackup run command I could confirm if it does full backups daily.

**Comment by salarali** [comment]
vmbackup-hourly runs every hour
```
/usr/local/bin/vmbackup \
	-storageDataPath=<path> \
	-snapshot.createURL=http://localst:8482/snapshot/create \
	-dst=s3://<bucket-name>/vm-metrics/node1/latest
```

vmbackup-daily runs every 24 hours. `YYYYMMDD` is incremented every run.
```
/usr/local/bin/vmbackup \
	-storageDataPath=<path> \
	-snapshot.createURL=http://localst:8482/snapshot/create \
	-dst=s3://<bucket-name>/vm-metrics/node1/YYYYMMDD
```

One question that I had was what should I set the ttls on s3 to be? My retention on vmstorage is get to 90 days.
Based on my understanding of this, `s3://<bucket-name>/vm-metrics/node1/latest` would keep on having older than 90 days data, as it gets deleted, unless I set a very strict TTL on the latest folder? Something like 24 hours would work on this case?

**Comment by zekker6** [comment]
> Based on my understanding of this, s3://<bucket-name>/vm-metrics/node1/latest would keep on having older than 90 days data, as it gets deleted, unless I set a very strict TTL on the latest folder? Something like 24 hours would work on this case?

vmbackup will remove unneeded parts as well, so `node1/latest` will always contain current snapshot of the data on hourly basis and there is no need to do additional cleanup.

Overall this setup looks good to me, should be working just fine with basic retention configuration.
